msg/messages.js

```diff
@@ -239,6 +239,9 @@ Blockly.Msg.CONTROLS_IF_ELSE_TITLE_ELSE = Blockly.Msg.CONTROLS_IF_MSG_ELSE;
 /// tooltip - Describes the 'else' subblock during [https://github.com/google/blockly/wiki/IfElse#block-modification if block modification].
 Blockly.Msg.CONTROLS_IF_ELSE_TOOLTIP = 'Add a final, catch-all condition to the if block.';
 
+/// block text - This is a statement that unary operator that executes the statements in %2 if %1 evaluates to true. Otherwise, the statements in %3 are executed.
```

Comment:
> This is a statement that unary operator that executes...

Huh?  I don't understand this.  Also, non-programer translators will definitely not understand this.  The target audience for this message is something like an English teacher at a school in Hungary.